app/controllers/practices_controller.rb

```diff
@@ -190,6 +190,20 @@ def update
         end
       end
 
+      # Crop practice creator avatar
+      if params[:practice][:practice_creators_attributes].present?
+        params[:practice][:practice_creators_attributes].each do |key, pc|
+          if pc[:crop_x].present? && pc[:crop_y].present? && pc[:crop_w].present? && pc[:crop_h].present? && pc[:id].present? && pc[:_destroy] == 'false'
```

Comment:
i'm wondering if we can abstract this out for the different things we need to crop?